src/Task01/AppTask01.java

```diff
@@ -0,0 +1,13 @@
+package Task01;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class AppTask01 {
+    public static void main(String[] args) {
+        String[] dataList = "checkLis/aper/corn/paper/corn/apricots/bread/cleaner/rice/vegetables/salt".split("/");
```

Comment:
name is not descriptive, what data? what this data try to describe, is bad naming